src/src_V2/modules/manager/financial/components/payables/edit/drawer-content/payment-history/PaymentHistory.jsx

```diff
@@ -0,0 +1,229 @@
+  { id: 'attachment', label: 'Attachment' },
+];
+
+const EditableTableCell = ({ rowValue, type, inputLabel }) => {
+  const [input, setInput] = useState(rowValue);
+
+  return (
+    <TableCellWrapper>
+      {(() => {
+        switch (type) {
+          case inputTypes.text:
+            return (
+              <TextField
+                label={inputLabel}
+                size="small"
+                variant="outlined"
+                value={input}
+                onChange={(e) => setInput(e.target.value)}
+              />
+            );
+          case inputTypes.price:
+            return <TextField label={inputLabel} size="small" variant="outlined" value={input} />;
+          case inputTypes.date:
+            return <TextField label={inputLabel} size="small" variant="outlined" value={input} />;
+          case inputTypes.select:
+            return (
+              <>
+                <InputLabel id="demo-simple-select-label">Type</InputLabel>
+                <Select
+                  labelId="demo-simple-select-label"
+                  id="demo-simple-select"
+                  value={rowValue}
+                  onChange={(e) => setInput(e.value)}
+                  autoWidth
+                  label="Type"
+                >
+                  <MenuItem value={'Cash'}>Cash</MenuItem>
+                  <MenuItem value={'Transfer'}>Transfer</MenuItem>
+                </Select>
+              </>
+            );
+          case inputTypes.file:
+            return <TextField label={inputLabel} size="small" variant="outlined" value={input} />;
+          default:
+            return <TooltipText text={input || ''} />;
+        }
+      })()}
+    </TableCellWrapper>
+  );
+};
+
+export default function PaymentHistory({ history }) {
+  const loading = false;
+  const [data, setData] = useState(history || []);
+
+  console.log(data);
+
+  const addPayment = useCallback(() => {
+    setData((d) => {
+      return [
+        {
+          invoiceId: '',
+          paidAmount: '',
+          paymentDate: '',
+          paymentType: 'Cash',
+          attachment: null,
+        },
+        ...d,
+      ];
+    });
+  }, [data]);
+
+  const AddPaymentBtn = useCallback(() => {
+    return (
+      <Button variant="contained" onClick={addPayment}>
+        + Add payment
+      </Button>
+    );
+  }, [addPayment]);
+
+  return (
+    <Accordion className={commonStyles.accordion} expanded={true}>
+      <AccordionSummary
+        aria-controls="panel1a-content"
+        id="panel1a-header"
+        className={`${commonStyles.accordionSummary} ${styles.accordionBox}`}
+      >
+        <Box className={styles.paymentBox}>
+          <Box
+            component="img"
+            alt="Payment History"
+            src={creditCard}
+            className={`${styles.cardImg} ${commonStyles.panelImg}`}
+          />
+          <Typography className={commonStyles.detailsTxt}>
+            Payment History{' '}
+            {history?.length && (
```

Comment:
if history.length will be 0 it will show 0 instead nothing, pls write like this:
`            {!!history?.length && (
`

Reply:
Maybe write history?.length > 0 ?